src/router/metadata/cap.rest.resource.template.ts

```diff
@@ -73,9 +73,8 @@ export function makeGenericResources(
     return resources;
 }
 
-export async function makeResource(resourceType: string, resource: Resource) {
+export async function makeResource(resourceType: string, resource: Resource, updateCreate: boolean) {

```

Comment:
The new parameter is not needed. This function can figure it out with `resource.persistence.updateCreateSupported `